apps/compose_apps.py

```diff
@@ -38,17 +41,23 @@ def __init__(self, name, app_dir, image_downloader_cls=DockerDownloader):
             self._image_downloader_cls = image_downloader_cls
 
             args = [self.DockerComposeTool, 'compose', '-f', self.ComposeFile, 'config']
-            cmd_exe(*args, cwd=self.dir)
+            if quiet:
```

Comment:
I think that's fine. I just thought about another option

```
if quite:
  args.append('--quite')

```

Reply:
> The publish script has to download all the per-arch container images

I thought about it too. It's just not necessary, just thought, we know a tag of images we build, i.e. < commit short hash>.